src/lib/index.d.ts

```diff
@@ -0,0 +1,100 @@
+/* tslint:disable */
+/* eslint-disable */
+
+/* auto-generated by NAPI-RS */
+
+export const enum Frequency {
```

Comment:
Could we create non-const versions of these enums in the wrapper to allow consumer projects to use `isolateModules`?

Reply:
Sure. But let's do it as a separate PR. I created an issue for this https://github.com/lsndr/rrule-rust/issues/147